sdk/src/main/java/com/endare/adhese/sdk/AdheseOptions.java

```diff
@@ -59,6 +64,15 @@ public String getAsURL() {
         }
 
         builder.append(String.format("/%s%s", AdheseParameter.COOKIE_MODE.getKey(), cookieMode.getValue()));
+        TreeSet<String> sortedKeys = new TreeSet<>(customParameters.keySet());
+        for (String key: sortedKeys) {
+            StringBuilder values = new StringBuilder();
+            for (String value: customParameters.get(key)) {
+                values.append(String.format("%s%s", value, ";"));
```

Comment:
code like this makes want to abuse you with Tropic Thunder quotes. don't add the first semi-colon, so you don't have to delete it later...

Reply:
I assume you mean the last one? I'll change the loop

Reply:
thumbs up icon